### PR TITLE
Fixed CI

### PR DESCRIPTION
**Pull request by Big-Shark**

### CODE REVIEW

**Comment by marcj** [comment]
What are you trying to fix with this PR? :)

**Comment by Big-Shark** [comment]
This PR combines 2 my commits, #1000 and #1040 and fixes them.
This PR it has a updated test system, but the old testing methods supported

File: circle.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ dependencies:
     - echo "memory_limit=520M" > ~/.phpenv/versions/$(phpenv global)/etc/conf.d/memlimit.ini
     - if [ "$CIRCLE_NODE_INDEX" = "1" ]; then ./tests/bin/setup.mysql.sh; fi
     - if [ "$CIRCLE_NODE_INDEX" = "2" ]; then ./tests/bin/setup.pgsql.sh; fi
-    - if [ "$CIRCLE_NODE_INDEX" = "3" ]; then ./tests/bin/setup.sqlite.sh; fi
```
Comment (marcj) [inline circle.yml]: no sqlite test anymore?

Comment (Big-Shark) [inline circle.yml]: https://circleci.com/gh/propelorm/Propel2/275
```
if [ "$CIRCLE_NODE_INDEX" = "3" ]; then ./tests/bin/setup.sqlite.sh; fi
./tests/bin/setup.sqlite.sh: 9: ./tests/bin/setup.sqlite.sh: [[: not found 
```
This script does not work for a long time, and the tests are working is good.
I figured if it didn't work, and the tests work, then I can delete it)

**Comment by Big-Shark** [comment]
@marcj We have any problem?

File: circle.yml (hunk 1 above)
Comment (motin) [inline circle.yml]: From http://propelorm.org/documentation/cookbook/working-with-test-suite.html
> ./tests/bin/setup.sqlite.sh just removes the file at tests/test.sq3 to trigger a re-creating by our test suite.

**Comment by staabm** [comment]
Could you squash the commits?

**Comment by Big-Shark** [comment]
@staabm done.

**Comment by staabm** [comment]
In my opinion we can drop php 5.4 in propel2 because it is no longer a maintained php version

**Comment by Big-Shark** [comment]
@staabm +1

**Comment by marcj** [comment]
Ok, very nice as is. thanks @Big-Shark!

**Comment by staabm** [comment]
@marcj what do you think about dropping 5.4 as suggested above?

**Pull request merged**